torch/_refs/__init__.py

```diff
@@ -87,8 +87,9 @@
     "eq",
     "float_power",
     # 'floor_divide', # requires floor
-    # 'fmax', # requires where
-    # 'fmod',
+    "fmax",  # requires where
```

Comment:
What's up with these comments? 

We have refs.where now, for example, but I thought we wanted to implement fmax as wrapping a prim, anyway?